### ISSUE TITLE
[BUDI-7364] SSO Microsoft to set for up in helm

### ISSUE THREAD

**Issue by SouzaGuilherme** [Budibase/budibase#11388]
**Describe the feature request**
A clear and concise description of what the feature request.

I believe it would be interesting to have the option to upload Microsoft's SSO by default to the hel as well, just as Google already uploads today. It would make it a lot easier!

Looking at the files, I believe it's pretty simple. Can I open a PR with the change?
**Screenshots**

<sub>[BUDI-7364](https://linear.app/budibase/issue/BUDI-7364/sso-microsoft-to-set-for-up-in-helm)</sub>

**Comment by shogunpurple** [comment]
@SouzaGuilherme yes, no problem at all - happy to review a PR with a contribution for this. Thanks in advance!
